.github/workflows/test.yml

```diff
@@ -49,23 +49,55 @@ jobs:
           - toxenv: py39-dj40
             python-version: 3.9
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

Comment:
Should you pull in the latest here? Which is 2.4.0 https://github.com/actions/checkout/releases

Reply:
Or does leaving out the minor and patch bring in the latest anyway?

Reply:
I desperately don't want to have to manage patch or minor versions of all of the actions across all our repos, and the [usage](https://github.com/actions/checkout#usage) does not suggest to specify anything other than major version.